feature-libs/product-configurator/rulebased/styles/_configurator-attribute-product-card.scss

```diff
@@ -97,6 +97,11 @@
           align-self: flex-end;
         }
       }
+
```

Comment:
The changes in this file are breaking and need to be documented in  /docs/migration/css-changes-in-version-4.md

Reply:
Done